Formula/rocksdb.rb

```diff
@@ -11,7 +12,6 @@ class Rocksdb < Formula
     sha256 "b058bea89af8825ab24bcebe61418ed20a8d504805292dda9d5addca181d13e1" => :yosemite
   end
 
-  option "without-lite", "Don't build mobile/non-flash optimized lite version"
   option "with-tools", "Build tools"
```

Comment:
You'd just remove it. We only use "deprecated_option" when it's being mapped to something other than nothing.